src/pages/open-source.js

```diff
@@ -21,12 +25,12 @@ const OpenSource = ({ data }) => {
     <Layout>
       <Head page={{ title, slug, seoTitle, seoMetaDescription }} />
       {/* <CaseStudyPreview caseStudy={} /> */}
+      <OpenDeliverables {...data} />
```

Comment:
If we're going to destructure everything going into these 2 sections, then we should probably be consistent and use the same pattern going into `PartnershipsSection`?

Reply:
There is a lot of props going into these two components, destructuring here and then again in both of the sections doesn't afford us anything.